examples/semi_supervised/plot_label_propagation_digits_active_learning.py

```diff
@@ -69,29 +72,36 @@
     pred_entropies = stats.distributions.entropy(
         lp_model.label_distributions_.T)
 
-    # select five digit examples that the classifier is most uncertain about
-    uncertainty_index = uncertainty_index = np.argsort(pred_entropies)[-5:]
+    # select up to 5 digit examples that the classifier is most uncertain about
+    uncertainty_index = np.argsort(pred_entropies)[::-1]
+    uncertainty_index = uncertainty_index[
+        np.in1d(uncertainty_index, unlabeled_indices)][:5]
 
     # keep track of indices that we get labels for
     delete_indices = np.array([])
 
-    f.text(.05, (1 - (i + 1) * .183),
-           "model %d\n\nfit with\n%d labels" % ((i + 1), i * 5 + 10), size=10)
+    # for more than 5 iterations, visualize the gain only on the first 5
+    if i < 5:
```

Comment:
I'm not sure this is necessary. Perhaps if we want the number of iterations to look like a parameter the user might play with, we should pull it out into a named variable.